src/main/java/org/folio/rest/impl/OrdersImpl.java

```diff
@@ -119,26 +116,27 @@ public void postOrders(String lang, CompositePurchaseOrder compPO, Map<String, S
   public void putOrdersById(String orderId, String lang, CompositePurchaseOrder compPO, Map<String, String> okapiHeaders,
       Handler<AsyncResult<javax.ws.rs.core.Response>> asyncResultHandler, Context vertxContext) {
     PutOrdersByIdHelper putHelper = new PutOrdersByIdHelper(okapiHeaders, asyncResultHandler, vertxContext, lang);
-    if (CollectionUtils.isEmpty(compPO.getPoLines())) {
-      putHelper.updateOrder(orderId, compPO);
+    if (org.apache.commons.lang.StringUtils.isEmpty(compPO.getPoNumber())) {
```

Comment:
This line can be removed as it will not be reachable. PO Number pattern validation will be handled by  @Pattern(regexp = "^[a-zA-Z0-9]{5,16}$") using @validate

Reply:
Since the number is an optional field, it may not be in the request. In this case, the user can expect the number to be deleted.

Reply:
ah good point , Thanks.